__tests__/components/ResourceCreation/TestResourceCreation.test.tsx

```diff
@@ -149,14 +149,22 @@ describe('TestResourceCreation', () => {
             resourceType: 'Procedure',
             id: 'test-id',
             status: 'completed',
-            subject: {}
+            subject: {},
+            code: {
+              coding: [
+                {
+                  code: '123',
+                  display: 'Colectomy'
+                }
+              ]
+            }
           }
         ]
       }
     });
     const MockSelectedPatient = getMockRecoilState(selectedPatientState, 'example-test-case');
 
-    render(
+    const baseDom = render(
```

Comment:
Do we need to set this as a const? I am getting an eslint warning since `baseDom` doesn't get called anywhere.